modules/videoio/src/cap.cpp

```diff
@@ -315,6 +316,19 @@ bool VideoCapture::open(int cameraNum, int apiPreference, const std::vector<int>
         CV_Error_(Error::StsError, ("could not open camera %d", cameraNum));
     }
 
+    for (size_t i = 0; i < deprecatedBackends.size(); i++)
+    {
+        const VideoBackendInfo& info = deprecatedBackends[i];
+        if (apiPreference == info.id)
+        {
+            //chosen backend is removed from OpenCV
+            CV_LOG_INFO(NULL, "VIDEOIO: chosen backend is removed from OpenCV");
+            return false;
```

Comment:
This message is not informative (it doesn't show even `id` or the `name`).
Code comment is redundant here.